frontend/src/pages/events/prayerGatheringEvents.js

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+
+const PrayerGatheringEvents = () => {
+  return (
+    <div className="bg-Neutral-200 w-full justify-center flex flex-col items-center py-[40px]">
+      <div className="flex-col justify-start items-center gap-y-2 flex mt-2">
+        <div className="subheading">PRAYER GATHERINGS</div>
+        <div className="text-black text-4xl font-semibold font-['Raleway'] leading-10">BY PRAYING, WE LEARN TO PRAY.</div>
```

Comment:
```suggestion
        <div className="text-black text-4xl font-semibold leading-10">BY PRAYING, WE LEARN TO PRAY.</div>
```

the default font is  Raleway.